src/packages/recompose/__tests__/nest-test.js

```
@@ -11,15 +11,10 @@ test('nest nests components from outer to inner', () => {
-      <A pass="through">
-        <B pass="through">
-          <C pass="through">Child</C>
-        </B>
-      </A>
-    )
-  ).toBe(true)
+  const { container } = render(<Nest pass="through">Child</Nest>)
+  // nest(A,B,C) creates nested divs, all with pass="through"
+  const divs = container.querySelectorAll('div')
+  expect(divs.length).toBe(3)
+  divs.forEach(div => expect(div.getAttribute('pass')).toBe('through'))
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)